src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- Big Search Form -->
+    <form class="search-form search-big"
+      data-qa="big"
+      action="/"
+      method="get"
+    >
```

Comment:
```suggestion
    <form
      class="search-form"
      data-qa="big"
      action="/"
      method="get"
    >
```
get rid of `big` selectors